deps/url.lua

```diff
@@ -126,6 +126,7 @@ local function parse(url, parseQueryString)
     search = search,
     query = query,
     auth = auth,
+    href = href,
     hash = hash
   }
   parsed.href = URL.format(parsed)
```

Comment:
Does this not overwrite the `href` set above?

Reply:
Yes, will fixit.